flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/expertgroup/ExpertGroupJpaRepository.java

```diff
@@ -1,13 +1,44 @@
 package org.flickit.assessment.data.jpa.kit.expertgroup;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 
+import java.util.List;
 import java.util.UUID;
 
 public interface ExpertGroupJpaRepository extends JpaRepository<ExpertGroupJpaEntity, Long> {
 
     @Query("SELECT e.ownerId FROM ExpertGroupJpaEntity as e where e.id = :id")
     UUID loadOwnerIdById(@Param("id") Long id);
+    @Query("""
+    SELECT
+        e.id as id,
+        e.name as name,
+        e.picture as picture,
+        e.bio as bio,
+        e.ownerId as ownerId,
+        COUNT(ak) as publishedKitsCount
+    FROM ExpertGroupJpaEntity e
+    LEFT JOIN AssessmentKitJpaEntity ak ON e.id = ak.expertGroupId AND ak.isActive = true
+    LEFT JOIN ExpertGroupAccessJpaEntity ac ON ac.expertGroupId = e.id
+    WHERE ac.userId = :currentUserId
+    GROUP BY e.id, e.name, e.picture, e.bio, e.ownerId
+""")
+
+
+
+    Page<ExpertGroupWithDetailsView> getExpertGroupSummaries(Pageable pageable,
```

Comment:
We usually put `pageable` as the last parameter.

Reply:
Done